### DISCUSSION TITLE
AdonisJS Discord Server?

### DISCUSSION THREAD

**Discussion by prescience-data** [adonisjs/core#1924] [Ideas]
I've noticed a lot of frameworks/packages with good communities have created Discord servers which I've found makes the interaction a lot more dynamic and personal.

Just throwing it out there to see if the maintainer team had any interest in setting one up! 😁

**Comment by mathieuh** [comment]
Hi @prescience-data,

There is one already https://discord.com/invite/vDcEjq6.

**Comment by darshithedpara** [comment]
Hi @thetutlage @RomainLanz 
I wonder why discord server isn't mentioned on the Version 5 site ? 
I've searched a lot and I couldn't find easily and later on I found on the Version 4 side inside on contributing guides section

You should add Discord Server link under Community section in the footer and header.
